src/frontend/src/pages/myfeeds.js

```diff
@@ -14,6 +14,37 @@ const useStyles = makeStyles(theme => ({
 
 export default function MyFeeds() {
   const classes = useStyles();
+  const [feedAuthor, setFeedAuthor] = useState('');
+  const [feedUrl, setFeedUrl] = useState('');
+
+  function handleAuthorChange(author) {
+    setFeedAuthor(author);
+  }
+
+  function handleUrlChange(url) {
+    setFeedUrl(url);
+  }
+
+  async function addFeed() {
+    try {
+      const response = await fetch(`${process.env.telescopeUrl}/feeds/`, {
```

Comment:
Here, `telescopeUrl` should be replaced with [`API_URL`](https://github.com/Seneca-CDOT/telescope/blob/master/env.example#L7) and provided with a [default value](https://github.com/Seneca-CDOT/telescope/issues/396).

For example, borrowing [code from gatsby-config.js](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/gatsby-config.js#L19):
```js
try {
  const telescopeUrl = process.env.API_URL || `http://localhost:${process.env.PORT || 3000}`;
  const response = await fetch(`${telescopeUrl}/feeds/`, {
```

Reply:
> 
> 
> Here, `telescopeUrl` should be replaced with [`API_URL`](https://github.com/Seneca-CDOT/telescope/blob/master/env.example#L7) and provided with a [default value](https://github.com/Seneca-CDOT/telescope/issues/396).
> 
> For example, borrowing [code from gatsby-config.js](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/gatsby-config.js#L19):
> 
> ```js
> try {
>   const telescopeUrl = process.env.API_URL || `http://localhost:${process.env.PORT || 3000}`;
>   const response = await fetch(`${telescopeUrl}/feeds/`, {
> ```

OK, after testing with these changes, I ran into the same issue, but now I believe I can concluded that the issue I posted about in https://github.com/Seneca-CDOT/telescope/pull/748#issuecomment-592180181 is a 'me problem' thanks to some additional information being logged:

![image](https://user-images.githubusercontent.com/13500769/75569374-f12f0580-5a22-11ea-8d94-3951d0839747.png)

I'm not sure why my Redis configuration is apparently misconfigured in this way, but I'll be looking into it... 🤔

Reply:
> I'm not sure why my Redis configuration is apparently misconfigured in this way, but I'll be looking into it... 🤔

Maybe this has something to do with it:
![image](https://user-images.githubusercontent.com/13500769/75569771-c3968c00-5a23-11ea-84f8-f7e2acbae736.png)

Reply:
After testing these changes, I'm unfortunately still running into the same issue. I captured more logs, which I hope can be used to help troubleshoot this issue (which I suspect is simply a 'me issue', stemming from a Redis misconfiguration, but we'll see):
- Telescope logging: https://pastebin.com/hR99m2h3
- Corresponding logging from [`redis-cli monitor`](https://redis.io/commands/monitor): https://pastebin.com/APbeGt84

Reply:
Agreed ^